tests/doghouse/test_snapshot.py

```
@@ -0,0 +1,100 @@
+"""Tests for Snapshot equivalence and serialization."""
+import datetime
+from doghouse.core.domain.blocker import Blocker, BlockerType, BlockerSeverity
+from doghouse.core.domain.snapshot import Snapshot
+
+
+def test_is_equivalent_same_state():
+    b = Blocker(id="t1", type=BlockerType.UNRESOLVED_THREAD, message="fix")
+    s1 = Snapshot(
+        timestamp=datetime.datetime(2026, 1, 1, tzinfo=datetime.timezone.utc),
```

Ruff (0.15.7)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)